Firebase/InAppMessaging/Data/FIRIAMFetchResponseParser.m

```diff
@@ -178,12 +178,16 @@ - (FIRIAMMessageDefinition *)convertToMessageDefinitionWithMessageDict:(NSDictio
 
     NSDictionary *content = (NSDictionary *)contentNode;
     FIRIAMRenderingMode mode;
-    UIColor *viewCardBackgroundColor, *btnBgColor, *btnTxtColor, *titleTextColor;
+    UIColor *viewCardBackgroundColor, *btnBgColor, *btnTxtColor, *secondaryBtnTxtColor,
+        *titleTextColor;
     viewCardBackgroundColor = btnBgColor = btnTxtColor = titleTextColor = nil;
 
-    NSString *title, *body, *imageURLStr, *actionURLStr, *actionButtonText;
-    title = body = imageURLStr = actionButtonText = actionURLStr = nil;
+    NSString *title, *body, *imageURLStr, *landscapeImageURLStr, *actionURLStr,
+        *secondaryActionURLStr, *actionButtonText, *secondaryActionButtonText;
+    title = body = imageURLStr = landscapeImageURLStr = actionButtonText =
+        secondaryActionButtonText = actionURLStr = secondaryActionURLStr = nil;
```

Comment:
This and the other mass-assignment above are both unnecessary since ObjC (ARC) initializes pointer types to nil automatically.

https://stackoverflow.com/questions/9071990/are-pointers-always-set-to-nil-on-declaration

Reply:
Ryan suggested I explicitly set to `nil` here-- kind of a matter of style but I'll err on the side of consistency (and not having to make another change).